src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -181,16 +185,21 @@ private static String getReasonForBeingLate(SyaCaseWrapper syaCaseWrapper) {
     }
 
     private static String getDwpIssuingOffice(SyaCaseWrapper syaCaseWrapper) {
+        DwpAddressLookupService dwpLookup = new DwpAddressLookupService();
         String benefitType = syaCaseWrapper.getBenefitType().getCode();
+
         if (benefitType.equalsIgnoreCase("uc")
                 && (syaCaseWrapper.getMrn() == null
                 || syaCaseWrapper.getMrn().getDwpIssuingOffice() == null)) {
             return "Universal Credit";
+
+        } else if (CARERS_ALLOWANCE == Benefit.getBenefitByCode(benefitType)) {
```

Comment:
Can see this method getting messy the more benefit types we add. Can it follow the same pattern as other benefit types (on line 202 below) or is that a no go?

Reply:
Confirmed with Hem, Carer's Allowance and UC will not have a issuing office so have tidied this as far as possible.